src/pages/confs/2017_jsconf_eu/preact_into_the_void.md

```diff
@@ -67,5 +67,6 @@ Related Links | Rating
 [HyperScript](https://github.com/hyperhype/hyperscript) | ★★★☆☆
 [IRHydra2](http://mrale.ph/irhydra/2/) | ★★★☆☆
 [ESBench](https://esbench.com/) | ★★★☆☆
-
-</article>
\ No newline at end of file
+[Video](https://www.youtube.com/watch?v=LY6y3HbDVmg) |
+[Slides](http://slides.com/developit/preact-into-the-void0#/) |
```

Comment:
Please, follow the rest of links for consistency. Missing the stars (hopefully a future feature).